content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -182,6 +182,16 @@ the `/var/log` directory, bypassing the default logging mechanism (the component
 do not write to the systemd journal). You can use Kubernetes' storage mechanisms
 to map persistent storage into the container that runs the component.
 
+Kubelet allows changing the pod logs directory from default `/var/log/pods`
+to a custom path. This adjustment can be made by configuring the `podLogsDir`
+parameter in the kubelet's configuration file.
+
+{{< caution >}}
```

Comment:
We should also mention that we only support the location being on the same disk as /var.

if one has the logs on a separate filesystem from /var then we will not tract that filesystem filling up.

Reply:
Updated 9f34a75c361c44929c5dba7615974d551d38fed2